tests/integration/charts/cluster_full_stack.yaml

```diff
@@ -26,8 +26,9 @@ medusa:
   storage: s3_compatible
 
   storage_properties:
-      port: 9000
-      secure: "False"
+    port: 9000
+    secure: "False"
+    multi_part_upload_threshold: 1024
```

Comment:
Will this setting be triggered in the tests? I don't see any test changes.

Reply:
it'll impact how Medusa deals with files larger than the threshold. They will use the "multi part upload" which behind the scenes relies on the provider's cli.
It was the missing piece to discover the Azure bug.